doc/api/vm.md

```diff
@@ -437,6 +437,9 @@ changes:
     `cachedData` property of the returned `vm.Script` instance.
     The `cachedDataProduced` value will be set to either `true` or `false`
     depending on whether code cache data is produced successfully.
+  * `resolveDynamicImport` {Function} See [`module.link()`][]
+    * `specifier` {string}
+    * `scriptOrModule` {Script | Module}
```

Comment:
nit: `{Script|Module}`

I'd also prefer `Script` and `Module` to be scoped to `vm.Script`/`vm.Module`, so that it doesn't conflict with `require('module').Module`.